### ISSUE TITLE
evaluation results for image quality metrics

### ISSUE THREAD

**Issue by Dreamer-HSX** [yifan123/flow_grpo#122]
For SD3.5-M evaluation of image quality metrics: Aesthetic, DeQA, ImgRwd, PickScore and UniRwd on DrawBench.
The results in the paper is: **5.39, 4.07, 0.87, 22.34, 3.33**, respectively.
I test SD3.5-M on dataset/drawbench/test.txt and got results: **5.40, 4.07, 0.83, 22.41, 3.08**, where results for ImgRwd, PickScore and UniRwd are different.

The core sampling code is:
```python
pipe = StableDiffusion3Pipeline.from_pretrained("stabilityai/stable-diffusion-3.5-medium/")
with torch.no_grad():
       image = pipe(
            prompt, 
            height=512,
            width=512,
            num_inference_steps=25,
            guidance_scale=4.5,
            # generator=generator
        ).images[0]
image.save(output_path)
```

The reward models are:
ImgRwd: https://huggingface.co/THUDM/ImageReward/blob/main/ImageReward.pt
PickScore: yuvalkirstain/PickScore_v1
UniRwd: CodeGoat24/UnifiedReward-7b-v1.5

I don't if I'm doing this wrong. Can you help me check this out? Thanks for the great work~😊
